tests/unit/testsuite.js

```diff
@@ -305,11 +305,11 @@ window.domEqual = function( selector, modifier, message ) {
 			result = {};
 		$.each( properties, function( index, attr ) {
 			var value = elem.prop( attr );
-			result[ attr ] = value !== undefined ? value : "";
+			result[ attr ] = value != null ? value : "";
```

Comment:
This strict comparison was fine as-is; `.prop` lacks the DOM semantics that urged us to try changing `.attr`.